pjsip/include/pjsua2/call.hpp

```diff
@@ -1051,8 +1051,8 @@ struct OnCallRxReinviteParam
     
     /**
      * Status code to be returned for answering the offer. On input,
-     * it contains status code 200. Currently, valid values are only
-     * 200 and 488.
+     * it contains status code PJSIP_SC_OK. Currently, valid values are only
+     * PJSIP_SC_OK and PJSIP_SC_NOT_ACCEPTABLE_HERE.
```

Comment:
Perhaps it's just me, having worked with SIP for so long, the number sticks more to me than the text. So, I'd suggest not to remove the number, i.e.
`PJSIP_SC_OK (200)` and `PJSIP_SC_NOT_ACCEPTABLE_HERE (488)`.

Also for other instances above and below.